api/api.go

```diff
@@ -437,6 +441,62 @@ type CloudBackupSchedEnumerateResponse struct {
 	Schedules map[string]CloudBackupScheduleInfo
 }
 
+type Operation string
+
+const (
```

Comment:
Define all of these in api.proto to make it easier when we add this support to the SDK. They will need to be enums. Check [SDK Requirements](https://github.com/libopenstorage/openstorage/blob/master/docs/dev-sdk.md#sdk-requirements) for enums requirements

Reply:
Don't want to define them as enum, want to use strings like the cloudbackup API types. Can't define enums in the proto file so keeping them here.

Reply:
I understand what you mean, but these values must be defined in api.proto and not here for when the SDK does call this.

Reply:
@jvinod mentioned we can still get the string of the enum from the protobuf defintion so moved it to the proto file